src/styles/clock.scss

```diff
@@ -0,0 +1,53 @@
+.clock {
+  position: relative;
+  height: $clock-size;
+  width: $clock-size;
+  margin: 50px;
+  background: radial-gradient(#505050, #303030);
+  border: 3px solid #302e2c;
+  box-shadow: 5px silver;
+
+  border-radius: 50%;
+
+  &__arrow {
+    position: absolute;
+    left: 50%;
+    top: 30px;
+    height: $arrow-h;
+    width: $arrow-w;
+    background-color: #171717;
+    transform-origin: 50% $top-space;
+    animation: rotate 3600s infinite linear;
+
+    &::after {
+      content: "^";
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 40px;
+      font-weight: 900;
+      line-height: 13px;
+      text-align: center;
+      margin-left: -9px;
+      color: #171717;
+    }
```

Comment:
you can use this guide (http://codeguide.academy/html-css.html#css-order) for formatting